iam.tf

```diff
@@ -163,4 +163,17 @@ resource "aws_iam_policy" "ssm_access_policy" {
 resource "aws_iam_role_policy_attachment" "ssm_access" {
   role       = aws_iam_role.lambda_application_execution_role.name
   policy_arn = aws_iam_policy.ssm_access_policy.arn
+}
+
+resource "aws_iam_policy" "custom_lambda_policy" {
+  count       = local.custom_policy_required ? 1: 0
+  name        = "LambdaApplication-${replace(var.application_name, "/-| |_/", "")}-CustomLambdaPolicy"
+  description = "Create custom lambda policy"
```

Comment:
@shikhaFNZC This description gets added to the policy resource, and it's not a great description. Ideally, this should be a variable to could be overridden, with a default value of something like 

```
A custom policy for the Lambda application module execution role
```

Reply:
There's still no variable for policy description. Can this be added like below
```
resource "aws_iam_policy" "custom_lambda_policy" {
  ...
  description = var.custom_policy_description
  ...
}
```
Then the variable could look like this

variable "custom_policy_description" {
  type        = string
  description = "Allows to override the custom Lambda policy's description"
  default     = "A custom policy for the Lambda application module execution role"
}